src/test/java/se/diabol/jenkins/pipeline/sort/LatestActivityComparatorTest.java

```diff
@@ -61,8 +61,8 @@ public void testCompare() {
         pipelinesB.add(pipelineB);
 
 
-        Component componentB = new Component("B", "B", "job/A", false, pipelinesB);
-        Component componentA = new Component("A", "A", "job/B", false, pipelinesA);
+        Component componentB = new Component("B", "B", "job/A", false, pipelinesB,3,false);
```

Comment:
extract false to a constant declared in the beginning of the class to declare the intent of the parameter, which then can be reused throughout the test file